pkg/osd_cluster_upgrader/config.go

```diff
@@ -64,6 +56,9 @@ func (cfg *osdUpgradeConfig) IsValid() error {
 	if cfg.NodeDrain.Timeout <= 0 {
 		return fmt.Errorf("Config nodeDrain timeOut is invalid")
 	}
+	if cfg.NodeDrain.WorkerNodeTime <= 0 {
+		return fmt.Errorf("Config maintenace workerNodeTime is invalid")
```

Comment:
```suggestion
		return fmt.Errorf("Config nodeDrain workerNodeTime is invalid")
```
(+ whatever it's renamed to)